update landing page (#455)
MDAnalysis/MDAnalysis.github.io: index.md

<img src="{{ site.baseurl }}/public/mdanalysis-logo_square.png"
style="float: right" alt="MDAnalysis" width="30%"/>

<!-- TODO: Correct link so it links to the specific section of this page [multiple formats]({{site.pypi.docs}}/documentation_pages/coordinates/init.html#id1)
 -->

> **MDAnalysis** is an open-source Python library for analyzing molecular dynamics (MD) trajectories across [multiple formats]({{site.pypi.docs}}/documentation_pages/coordinates/init.html#id1). It enables seamless reading and writing of simulation data, allowing users to efficiently analyze molecular structures and dynamics, including particle-based trajectories and individual coordinate frames (e.g., biomolecules in the [PDB format][]). 

Reviewer: I know that this is the rewrite of the original page, but it seems a bit narrow (it was written 10+ years ago...). I feel we should start bigger with the MDAnalysis Project and then mention the MDAnalysis library itself.

Reviewer: The second sentence should be "The MDAnalysis Project is a fiscally sponsored project of NF, a ....".

> With MDAnalysis, you can access atomic coordinates as [NumPy][] arrays, providing a flexible and efficient framework for complex analysis tasks. The library includes powerful [atom selection commands]({{site.pypi.docs}}/documentation_pages/selections.html) for extracting subsets of structures, supports trajectory transformations (e.g., fitting to a reference structure), and can write out trajectories along with [atom selections]({{site.pypi.docs}}/documentation_pages/selections_modules.html#selection-exporters) in formats suitable for visualization or native analysis tools.

Reviewer: leave out selection exporters, they are not that interesting to most people

Reviewer: I think the UG has better pages for selections and also for file formats?

> Learn more about our **mission, development, team, and governance** on the [About MDAnalysis]({{ site.baseurl }}about/) page.

Reviewer: / after {{site.baseurl}} ?

## Get Started  

If you're new to MDAnalysis, explore the following resources to install the library, access tutorials, and dive into the documentation:

- [Getting Started]({{ site.baseurl }}/pages/getting_started/)  
- [Learning MDAnalysis]({{ site.baseurl }}/pages/learning_MDAnalysis)  
- [Documentation]({{ site.baseurl }}/pages/documentation)  

> ## Community Hub

Reviewer: What does "Hub" mean here?

MDAnalysis is driven by an active **community of users and contributors**. Stay updated and get involved through the following pages:  

<!-- TODO: Add link [Get Involved]({{ site.baseurl }}/pages/about/#get-involved)
 -->
- [Community]({{ site.baseurl }}/pages/community/) &mdash; Get involved, ask questions, and collaborate.
- [Blog]({{ site.baseurl }}/blog) &mdash; Stay updated with development news and community highlights.
- [Events]({{ site.baseurl }}/pages/events/) &mdash; Join workshops, conferences, and mentoring programs.

## Contributing 

Want to contribute to MDAnalysis? Here’s how:  

- [Contribute]({{ site.baseurl }}/pages/contribute/) &mdash; Learn how to help improve MDAnalysis through coding, documentation, or discussions.  
- [MDAKits and MDA-based tools]({{ site.baseurl }}/pages/mdakits/) &mdash; Build and extend MDAnalysis with MDAKits.  

## Citing MDAnalysis

If you use MDAnalysis in your research, please cite it appropriately and consider displaying our MDAnalysis badge in your projects. All citation formats and badge instructions are available in our [Citation guidelines]({{ site.baseurl }}/pages/citations/).

## Funding & Support

> MDAnalysis has received generous support from organizations such as the **Chan Zuckerberg Initiative, Google, NumFOCUS, and the National Science Foundation**.  

Reviewer 2: I would mention 'numfocus sponsored project' before support from any other grants

Reviewer: I think it should be in the 1st paragraph of the page.

> To learn more about out Sponsors or how to support MDAnalysis, visit our [Funding]({{ site.baseurl }}/pages/funding/) page.

Reviewer 2: I would keep the support button you deleted above in this section. normally we want to avoid repetition, but I can't see making it easy to donate to us a bad thing:) ideally there could be a dedicated 'Donate' button in the black side bar.

> Additionally, consider displaying our badge in your projects that use

Reviewer: Why not keep it here, too?

<img src="{{ site.baseurl }}/public/mdanalysis-logo_square.png"
style="float: right" alt="MDAnalysis" width="30%"/>

>  The **MDAnalysis Project** develops open-source tools for the analysis of molecular simulation data, providing researchers with efficient and accessible solutions for studying molecular structures and dynamics. The project is a fiscally sponsored initiative of [NumFOCUS][], a nonprofit organization that supports open-source scientific computing.

Reviewer: "initiative" is not the correct wording. You need to use the language from https://numfocus.org/sponsored-projects :

MDAnalysis is a fiscally sponsored project of NumFOCUS, a nonprofit that promotes open practices in research, data, and scientific computing.

(I took the last part from their https://numfocus.org/community/mission page)
```suggestion
The **MDAnalysis Project** develops open-source tools for the analysis of molecular simulation data, providing researchers with efficient and accessible solutions for studying molecular structures and dynamics.
MDAnalysis is a fiscally sponsored project of [NumFOCUS][], a nonprofit that promotes open practices in research, data, and scientific computing.

```
I also inserted a paragraph break.

> At the core of the project is MDAnalysis, **an open-source Python library** for analyzing molecular dynamics (MD) trajectories across [multiple formats]({{ site.docs.userguide.url }}/formats/index.html#formats). MDAnalysis enables seamless reading and writing of simulation data, allowing users to efficiently analyze molecular structures and dynamics, including particle-based trajectories and individual coordinate frames (e.g., biomolecules in the [PDB format][]).

Reviewer: Let's typeset the library in monospaced font:
```suggestion
At the core of the project is `MDAnalysis`, **an open-source Python library** for analyzing molecular dynamics (MD) trajectories across [multiple formats]({{ site.docs.userguide.url }}/formats/index.html#formats). `MDAnalysis` enables seamless reading and writing of simulation data, allowing users to efficiently analyze molecular structures and dynamics, including particle-based trajectories and individual coordinate frames (e.g., biomolecules in the [PDB format][]).
```
This helps with distinguishing the project and the library. See our Style Guide https://github.com/MDAnalysis/branding/blob/main/STYLE_GUIDE.md

Reviewer: Actually, on re-reading the Style Guide carefully, I was wrong: We would NOT monospace MDAnalysis here because we are referring to the library by name, not the package itself. To stay consistent with the current Style Guide **ignore the comment suggestion above**.

You _could_ italicize *MDAnalysis* but that's optional — up to you if you think that this will make things clearer.

> With MDAnalysis, you can access atomic coordinates as [NumPy][] arrays, providing a flexible and efficient framework for complex analysis tasks. The library includes powerful [atom selection commands]({{ site.docs.userguide.url }}/selections.html) for extracting subsets of structures and supports trajectory transformations (e.g., fitting to a reference structure).

Reviewer: Follow [Style Guide](https://github.com/MDAnalysis/branding/blob/main/STYLE_GUIDE.md) and monospace `MDAnalysis`.

Reviewer: Actually, on re-reading the Style Guide carefully, I was wrong: We would NOT monospace MDAnalysis here because we are referring to the library by name, not the package itself. To stay consistent with the current Style Guide **ignore the comment suggestion above**.

You _could_ italicize *MDAnalysis* but that's optional — up to you if you think that this will make things clearer.

Learn more about our **mission, development, team, and governance** on the [About MDAnalysis]({{ site.baseurl }}/about/) page.

## Get Started  

> If you're new to MDAnalysis, explore the following resources to install the library, access tutorials, and dive into the documentation:

Reviewer: Follow [Style Guide](https://github.com/MDAnalysis/branding/blob/main/STYLE_GUIDE.md) and monospace `MDAnalysis`.

Reviewer: Actually, on re-reading the Style Guide carefully, I was wrong: We would NOT monospace MDAnalysis here because we are referring to the library by name, not the package itself. To stay consistent with the current Style Guide **ignore the comment suggestion above**.

You _could_ italicize *MDAnalysis* but that's optional — up to you if you think that this will make things clearer.

We _could_ write
```suggestion
If you're new to MDAnalysis, explore the following resources to install the `MDAnalysis` package, access tutorials, and dive into the documentation:
```
to make clear that this is the installable package that contains the code.

- [Getting Started]({{ site.baseurl }}/pages/getting_started/)  
- [Learning MDAnalysis]({{ site.baseurl }}/pages/learning_MDAnalysis)  
- [Documentation]({{ site.baseurl }}/pages/documentation)  

## Community 

MDAnalysis is driven by an active **community of users and contributors**. Stay updated and get involved through the following pages:  

<!-- TODO: Add link [Get Involved]({{ site.baseurl }}/pages/about/#get-involved)

>  -->

Reviewer 3: Is this a TODO meant to remain after the PR is merged?

If you're new to MDAnalysis, explore the following resources to install the library, access tutorials, and dive into the documentation:

- [Getting Started]({{ site.baseurl }}/pages/getting_started/)  
- [Learning MDAnalysis]({{ site.baseurl }}/pages/learning_MDAnalysis)  
- [Documentation]({{ site.baseurl }}/pages/documentation)  

## Community 

MDAnalysis is driven by an active **community of users and contributors**. Stay updated and get involved through the following pages:  

<!-- TODO: Add link [Get Involved]({{ site.baseurl }}/pages/about/#get-involved)
 -->
- [Community]({{ site.baseurl }}/pages/community/) &mdash; Get involved, ask questions, and collaborate.
- [Blog]({{ site.baseurl }}/blog) &mdash; Stay updated with development news and community highlights.
- [Events]({{ site.baseurl }}/pages/events/) &mdash; Join workshops, conferences, and mentoring programs.

## Contributing 

(Donations are made through [our fiscal sponsor][], [NumFOCUS][], which is
a 501(c)(3) non-profit charity in the United States; as such,
donations to NumFOCUS are tax-deductible as allowed by law.  As with
any donation, you should consult with your personal tax adviser or the

> IRS about your particular tax situation.)

Reviewer: Should remain next to donate button.

Reviewer: YOu can enclose in `<small>` tags or a CSS to decrease font size, or make it a footnote, but this information should be somewhere near the donate button.

> {{ site.numfocus.donate_button }}

Reviewer: Donation needs context — whatever we had there before.

Could also be under Contributing – you can contribute by giving us money (eg if you are a charity or company).

> To learn more about our Sponsors or how to support MDAnalysis, visit our [Funding]({{ site.baseurl }}/pages/funding/) page.

Reviewer: Maybe financial support should also be under Contributing.
